src/main/kotlin/org/phoenixframework/Transport.kt

```diff
@@ -155,6 +155,7 @@ class WebSocketTransport(
 
   override fun onClosing(webSocket: WebSocket, code: Int, reason: String) {
     this.readyState = Transport.ReadyState.CLOSING
+    webSocket.close(code, reason)
```

Comment:
Could you please also modify the `WebSocketTransportTest` to include this please?

```
    internal fun `sets ready state to CLOSING`() {
      ...

      transport.onClosing(mockWebSocket, 10, "reason")
      verify(mockWebSocket).close(10, "reason")
      ...
    }
```